package.json

```diff
@@ -38,31 +38,33 @@
     "@commitlint/config-angular": "8.2.0",
     "@commitlint/prompt": "8.2.0",
     "@commitlint/prompt-cli": "8.2.0",
-    "@edx/frontend-build": "^2.0.4",
-    "@edx/frontend-platform": "1.1.14",
-    "@edx/paragon": "7.1.5",
+    "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
+    "@edx/frontend-build": "5.4.0",
```

Comment:
Upgrading edx/frontend-build to be able to use module.config.js overrides. The upgrade resulted in the eslint config changing, and having to resolve minor ESLint errors/warnings.